lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -36,6 +36,8 @@ class DetailsRenderer {
         return this._renderText(details);
       case 'cards':
         return this._renderCards(/** @type {!DetailsRenderer.CardsDetailsJSON} */ (details));
+      case 'code':
```

Comment:
how do you feel about calling this something like `codeSnippet`? It feels like `code` will be a more primitive renderer more similar to `url` or `text`...just a `<pre>` and maybe (someday) some syntax highlighting or something